Rebus.AdoNet.Tests/SagaPersisterTests.cs

```diff
@@ -874,34 +1056,29 @@ public void NoChangesAreMadeWhenUoWIsNotCommitted()
 		public void FindSameSagaTwiceThrowsOnNoWait()
 		{
 			// Skip this test on sqlite, as it does not support locking.
-			if (ProviderName == GetSqliteProviderName())
+			if (!_features.HasFlag(Feature.NoWait))
 			{
-				Assert.Ignore("Test not supported on sqlite database.");
+				Assert.Ignore("Test not supported when NoWait Locking is disabled or unsupported.");
 				return;
 			}
 
-			var persister = CreatePersister(createTables: true, useLocking: true, useNoWaitLocking: true);
+			var persister = _persister;
 			var savedSagaData = new MySagaData();
 			var savedSagaDataId = Guid.NewGuid();
 			savedSagaData.Id = savedSagaDataId;
 			persister.Insert(savedSagaData, new string[0]);
 
 			using (_manager.Create())
+			using (_manager.GetScope(autocomplete: true))
 			{
 				persister.Find<MySagaData>("Id", savedSagaDataId);
 
 				Assert.Throws<AdoNetSagaLockedException>(() =>
 				{
-				//	using (var thread = new CrossThreadRunner(() =>
-				//	{
-						using (EnterAFakeMessageContext())
-						{
-							persister.Find<MySagaData>("Id", savedSagaDataId);
-						}
-				//	}))
-				//	{
-				//		thread.Run();
-				//	}
+					using (EnterAFakeMessageContext())
```

Comment:
Leave the commented code as it was. It was left there on purpose, as sometimes in order to debug some issues we do enable CrossThread running.

Reply:
ok